core/tests/protractor_desktop/additionalEditorAndPlayerFeatures.js

```diff
@@ -500,7 +500,7 @@ describe('Full exploration editor', function() {
     });
 
     // Refreshing to prevent stale elements after backing from previous page.
-    browser.refresh();
+    browser.driver.navigate().refresh();
```

Comment:
This is the main point of the fix -- I tried a different type of refresh: https://github.com/angular/protractor/issues/1143#issuecomment-95326669